openhtf/io/output/upload_json.py

```diff
@@ -0,0 +1,72 @@
+"""Module for uploading JSON-formatted test record files to server."""
```

Comment:
can we change the wording on this a little, so that instead of 'upload to server', it's 'HttpUpload'?  i think the 'to server' part is implied, but the upload mechanism is ambiguous.